test/runner.test.js

```diff
@@ -0,0 +1,12 @@
+const assert = require("assert").strict;
+const test = require("../dist/runner");
```

Comment:
I don't love hardcoding the path to `dist`. Might be worth looking at a better way.